daemons/deploy_aws_elasticsearch.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function create_elasticsearch_domain() {
+    echo Creating Elasticsearch domain with name: $elasticsearch_domain_name
+    # Just create a small ES cluster for now.
+    # It is easy to increase the capacity as needed.
+    elasticsearch_version=5.3 # Latest version supported by AWS as of 6/2017
+    elasticsearch_cluster_config=InstanceType=t2.small.elasticsearch,InstanceCount=1
+    # 35GB is maximum allowed for a t2.small.elasticsearch
+    ebs_options=EBSEnabled=true,VolumeType=gp2,VolumeSize=35
+
+    aws es create-elasticsearch-domain \
+      --domain-name $elasticsearch_domain_name \
+      --elasticsearch-version $elasticsearch_version \
+      --elasticsearch-cluster-config $elasticsearch_cluster_config \
+      --ebs-options $ebs_options
+}
+
+function update_elasticsearch_domain_policy() {
+    echo Updating policy for Elasticsearch domain with name: $elasticsearch_domain_name
+    sed "s/ELASTICSEARCH_DOMAIN/$elasticsearch_domain_name/g" ./es_policy.json.template > ./es_policy.json
```

Comment:
Generally it's preferable to use `$(dirname $0)/` instead of relative paths (`./`) to make sure the script can run when invoked from other directories.